src/HackUnlock/level1.js

```diff
@@ -0,0 +1,20 @@
+/* exported HULevel1 */
```

Comment:
I don't think the HackUnlock will ever get more than one level, so this feels an unnecessary abstraction at that point, and could be merged back with the controlpanel itself.

Reply:
I kind of like the consistency, but this makes sense.